src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const Robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return (
+        `name: ${this.name},`
+        + ` chip version: ${this.version}, wheels: ${this.wheels}`
+      );
+    },
+
+    goForward(move = 1) {
```

Comment:
let's give our variable more meaningful name
```suggestion
    goForward(step = 1) {
```